README.md

```diff
@@ -1 +1,210 @@
-# -hello-microvers
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+ 
+ 
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [-hello-microvers] <a name="about-project"></a>
+
+> This project was part of Microverse week one tasks.
+It gives the student an opportunity to learn and do with HTML, CSS, and Github.
+
+**[Hello Microvers]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Linters
+### Tech Stack <a name="tech-stack"></a>
+
+> The tech stack used in this project include HTML,CSS and Github.
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project contain the features listed below.
+
+- **[HTML]**
+- **[CSS]**
+- **[Github]**
+- **[Linters]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
```

Comment:
[OPTIONAL]
- [ ] Kindly remove these comments since it was just there for your guidance, keeping them will make your `README` look unprofessional.